### PR TITLE
Ignore watch task

### LINKED ISSUE #51: Ignore some tasks
Some tasks are not really worth viewing time for, such as Watch which may be left running and would always be the lion's share of time.

### PR DESCRIPTION
**Pull request by marlowpayne**
Fixes #51 

Related to #72

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -111,6 +111,12 @@ module.exports = function (grunt, cb) {
 		var tableDataProcessed = tableData.map(function (row) {
 			var avg = row[1] / totalTime;
 
+			// Issue 51: Ignore watch task, or tasks beginning with 'watch:'
+			var ignoreTask = 'watch';
+			if (!grunt.option('verbose') && (row[0] === ignoreTask || row[0].lastIndexOf(ignoreTask + ':', 0) === 0)) {
```
Comment (sindresorhus) [inline index.js]: Can be simplified:

```js
if (!grunt.option('verbose') && /^watch($|:)/.test(row[0])) {
```

File: index.js [hunk 2]
```diff
@@ -111,6 +111,12 @@ module.exports = function (grunt, cb) {
 		var tableDataProcessed = tableData.map(function (row) {
 			var avg = row[1] / totalTime;
 
+			// Issue 51: Ignore watch task, or tasks beginning with 'watch:'
```
Comment (sindresorhus) [inline index.js]: Change it to

> // hide the watch task

File: readme.md [hunk 3]
```diff
@@ -24,6 +24,8 @@ module.exports = function (grunt) {
 }
 ```
 
+**Note:** The 'watch' task (and tasks beginning with 'watch:') are ignored by default. They may be included by using `grunt --verbose`.
```
Comment (sindresorhus) [inline readme.md]: > and tasks beginning with 'watch:'

This is moot. Remove.

**Comment by marlowpayne** [comment]
Thanks for the review @sindresorhus. Changes have been made.

**Comment by sindresorhus** [comment]
Thanks :)

**Pull request closed without merging**